code/solutions/linting-and-testing/example_sqs_queue.yaml

```diff
@@ -0,0 +1,23 @@
+AWSTemplateFormatVersion: "2010-09-09"
+
+Description: AWS CloudFormation workshop - linting and testing - example SQS queue (uksb-1q9p31idr)
```

Comment:
```suggestion
Description: AWS CloudFormation workshop - Linting and testing (uksb-1q9p31idr)
```

Reply:
Please rename the file to `sqs-queue.yaml`. We should avoid naming files and variables with words such as `example`, `my` etc.